client/modules/core/main.js

```diff
@@ -135,17 +165,18 @@ export default {
       } else {
         permissions = checkPermissions;
       }
+
       // if the user has owner permissions we'll always check if those roles are enough
       // By adding the "owner" role to the permissions list, we are making hasPermission always return
       // true for "owners". This gives owners global access.
       // TODO: Review this way of granting global access for owners
       permissions.push("owner");
       permissions = _.uniq(permissions);
 
-      //
-      // return if user has permissions in the group
-      //
-      if (Roles.userIsInRole(userId, permissions, group)) {
+      // if accountPermissions includes any of checkPermissions, then we return true
+      // (this replaces `Roles.userIsInRole`)
+      const hasPermission = accountPermissions[permissionsGroup].some((permission) => checkPermissions.includes(permission));
+      if (hasPermission) {
```

Comment:
If has permission is always going to be a boolean, you can just return it and remove the `conditional and the return true/false`

Reply:
Also, I get an error with `checkPermissions.includes` when trying to edit a product

```
main.js:178 Uncaught TypeError: Cannot read property 'includes' of undefined
    at main.js:178
    at Array.some (<anonymous>)
    at roleCheck (main.js:178)
    at Object.hasPermission (main.js:222)
    at composer (tagListContainer.js:250)
    at tracker.js:40
    at Computation._compute (tracker.js:308)
    at new Computation (tracker.js:206)
    at Object.Tracker.autorun (tracker.js:576)
    at tracker.js:38
```

Reply:
Good call, think we were using the bool in a different way before and i just left it as close as I could.

Also fixed the `var` i was checking against, should have been `permissions`, which is built off `checkPermissoins`, instead of `checkPermissions`